src/conversion/model-selection-to-view-converters.js

```diff
@@ -200,12 +200,34 @@ function wrapCollapsedSelectionPosition( modelSelection, viewSelection, viewElem
 	}
 
 	let viewPosition = viewSelection.getFirstPosition();
+
+	// This hack is supposed to place attribute element *after* all ui elements if the attribute element would be
+	// the only non-ui child and thus receive a block filler.
```

Comment:
This comment tells what you do, but not why you do it. At least link to the issue is needed.